src/App.test.tsx

```diff
@@ -1,9 +1,33 @@
-import { render, screen } from "@testing-library/react"
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { cleanup, render, screen, waitFor } from "@testing-library/react"
 import App from "./App"
 import React from "react"
+import { usePlayers } from "./hooks/api/usePlayers"
+
+jest.mock("./hooks/api/usePlayers")
+
+const usePlayersMock = jest.mocked(usePlayers)
+
+beforeEach(() => {
+  jest.resetAllMocks()
+})
+
+afterEach(cleanup)
+
+const usePlayersReturn = jest.mocked({
+  hasCurrentSession: jest.fn(),
+  startPollPlayers: jest.fn(),
+})
+
+test("renders register page", async () => {
+  const promise = Promise.resolve(false)
```

Comment:
Create a promise that we can wait for, else the dreaded change outside act error is thrown.